templates/room-chat.qtpl

```diff
@@ -135,35 +150,15 @@
                     url = thumbUrl
                 }
 
-                mxc := mxclient.NewMXCURL(url, p.MediaBaseURL)
-                mxcThumbURL := mxc.ToThumbURL(360, 360, "scale")
-                alt := Str(ev.Content["body"])
+                mxcThumbURL := mxclient.NewMXCURL(url, p.MediaBaseURL).ToThumbURL(360, 360, "scale")
             %}
-            <a href="{%s mxc.ToURL() %}" rel="noopener">
-                <img class="m.image" src="{%s mxcThumbURL %}" alt="{%s alt %}" />
-                <br>
-                <sup>{%s Str(ev.Content["body"]) %}</sup>
-            </a>
+            {%= p.renderFileEvent(ev, mxcThumbURL) %}
+        {% case "m.video" %}
```

Comment:
If nothing is in these case statements does it drop down to the next case statement automatically?

Reply:
It does, no explicit `fallthrough` sadly

Reply:
Actually the go code it generates lacks the fallthrough so seems like it'll be wrong. Good catch

Reply:
sadly looks like it has no fallthrough support, so had to duplicate some lines